src/constants.ts

```diff
@@ -21,7 +21,8 @@ export type DIFFS =
 /** All mods to require/suggest. */
 export type MODS =
     "Chroma" |
-    "Noodle Extensions"
+    "Noodle Extensions" |
+    "Cinema"
```

Comment:
Something should enforce this so it's only allowed in `suggestions`

Reply:
👍